docs/api-bridging/spec.md

```diff
@@ -0,0 +1,1410 @@
+
+The Desktop Agent Bridge MAY support channel mapping ability to deal with channel differences that can arise.
+
+A key responsibility of the DAB is ensuring that the channel state of the connected agents is kept in-sync, which requires an initial synchronization step as part of the connection protocol.
+
+#### Bridging Desktop Agent on Multiple Machines
+
+As the bridge binds its websocket on the loopback address (127.0.0.1) it cannot be connected to from another device. Hence, an instance of the standalone bridge may be run on each device and those instances exchange messages in order to implement the bridge cross-device.
+
+However, cross-machine routing is an internal concern of the Desktop Agent Bridge, with each desktop agent simply communicating with a bridge instance located on the same machine. The connection protocol between bridges themselves is implementation specific and beyond the scope of this standard. Further, as FDC3 only contemplates interoperability between apps for a single user, it is expected that in multi-machine use cases each machine is being operated by the same user. However, methods of verifying the identity of user are currently beyond the scope of this Standard.
+
+### Connection Protocol
+
+On connection to the bridge's websocket, a handshake must be completed that may include an authentication step before a name is assigned  to the desktop agent for use in routing messages. The purpose of the handshake is to allow:
+
+* The desktop agent to confirm that it is connecting to an FDC3 Desktop Agent Bridge, rather than another service exposed via a websocket.
+* The bridge to require that the desktop agent authenticate itself, allowing it to control access to the network of bridged desktop agents.
+* The desktop agent to request a particular name by which it will be addressed by other agents and for the bridge to assign the requested name, after confirming that no other agent is connected with that name, or a derivative of that name if it is already in use.
+
+The bridge is ultimately responsible for assigning each desktop agent a name and for routing messages using those names. Desktop agents MUST accept the name they are assigned by the bridge.
+
+#### Step 1. Handshake
+
+Exchange standardized hello messages that identify:
+
+* That the server is a bridge, including:
+  * implementation details for logging by DA.
+  * supported FDC3 version(s).
+* That the client is an FDC3 DA, including:
+  * implementation details (ImplementationMeta returned by fdc3.getInfo() call) for logging by DA and sharing with other DAs.
+    * already includes supported FDC3 version.
+  * request for a specific agent name.
+
+```mermaid
+sequenceDiagram
+    participant DA as Desktop Agent A
+    participant DAB as Desktop Agent Bridge
+    participant DB as Desktop Agent B
+    participant DC as Desktop Agent C
+    DAB ->>+ DA: hello

```

Comment:
should we add the initial websocket connection to the flow diagram? currently it looks like it's the DAB connecting to the desktop agent